<launch>
    <node pkg="cv_camera" type="cv_camera_node" name="pioneer_max_camera">
        <param name="device_id" value="0"/>
        <param name="image_width" value="2592"/>
        <param name="image_height" value="1944"/>
    </node>
    <node pkg="gs_camera" type="photo_node.py" name="photo_node">
        <param name="image_topic" value="pioneer_max_camera/image_raw"/>
        <param name="folder_path" value="/home/ubuntu/photo"/>
    </node>
</launch>